legal-api/report-templates/template-parts/certificate-of-name-change/logo.html

```diff
@@ -0,0 +1,3 @@
+<div class="column-logo">
```

Comment:
Again, is this file (logo) specific to the Certif of Name Change, or it common?

Reply:
Yes, the logo is used by other certificates.